Sync with current state on Weblate

sjentsch opened the pull request:
Added German (complete), some Norwegian and some Slovenian (again by Matej U).

sjentsch committed:
Added translation using Weblate (German)

sjentsch committed:
Added translation using Weblate (Norwegian Bokmål)

sjentsch committed:
Added translation using Weblate (Slovenian)

sjentsch committed:
Translated using Weblate (German)

Currently translated at 33.8% (447 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/de/

sjentsch committed:
Translated using Weblate (German)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/de/

sjentsch committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/pt/

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 0.1% (1 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/sl/

sjentsch committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 0.1% (2 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/nb_NO/

sjentsch committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 1.0% (14 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/nb_NO/

sjentsch committed:
Weblate translations (German, some Norwegian, some Slovenian) - 2022-02-22

sjentsch committed:
Weblate translations (German, some Norwegian, some Slovenian) - 2022-02-22

sjentsch committed:
Weblate translations (German, some Norwegian, some Slovenian) - 2022-02-22

sjentsch committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 9.0% (119 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/nb_NO/

sjentsch committed:
Translated using Weblate (German)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/de/

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 2.6% (35 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/sl/

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 3.2% (43 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/sl/

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 27.2% (359 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/sl/

weblate committed:
Added translation using Weblate (Croatian)

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 36.8% (486 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/sl/

sjentsch committed:
Translated using Weblate (German)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/de/

sjentsch committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 24.4% (322 of 1319 strings)

Translation: jamovi/i18n_jmv
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv_i18n/nb_NO/

weblate committed:
Added translation using Weblate (Polish)

sjentsch committed:
Merge branch 'jamovi:main' into main

sjentsch committed:
Changed language code for Norwegian

jonathon-love commented on no.po:
```diff
@@ -0,0 +1,7947 @@
+"Last-Translator: Sebastian Jentschke <sebastian.jentschke@uib.no>\n"
+"Language-Team: Norwegian Bokmål <https://hosted.weblate.org/projects/jamovi/"
+"jmv_i18n/nb_NO/>\n"
+"Language: nb_NO\n"
```
don't forget this one here ... i'm wondering/hoping that `\n` will be ok.

sjentsch replied:
Can you figure it out (try whether they can be applied to jamovi). It is in the header so it actually shouldn't matter.
Otherwise, I have to find out whether there is a way to tweak that on Weblate but that might not be too easy.

jonathon-love replied:
the way it's currently written, it works off the value of this field (rather than the filename)

sjentsch replied:
Do you mean the entry in the "Language:"-field? If so, can you change manually this time (or tell me to do it) and merge the pull request?
The problem is that Weblate is set up to pull from this repo (jamovi/jmv_i18n) so it may change the entries once the file name is updated in the repo.

sjentsch replied:
I just did the change.

sjentsch committed:
House-cleaning for Norwegian, German and Portugese (make jamovi-repository and Weblate match up again)

sjentsch committed:
Changed Norwegian language code (nb_NO -> no)

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 40.9% (540 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/sl/

sjentsch committed:
Translated using Weblate (Spanish)

Currently translated at 98.0% (1293 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/es/

sjentsch committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 33.0% (436 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/nb_NO/

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 43.2% (570 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/sl/

jonathon-love commented:
i've committed the german and norwegian.

weblate committed:
Added translation using Weblate (Italian)

MAgojam committed:
Translated using Weblate (Italian)

Currently translated at 0.0% (0 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/it/

MAgojam committed:
Translated using Weblate (Italian)

Currently translated at 68.4% (903 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/it/

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 51.6% (681 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/sl/

MAgojam committed:
Translated using Weblate (Italian)

Currently translated at 70.8% (935 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/it/

sjentsch committed:
Merge branch 'jamovi:main' into main

sjentsch committed:
Merge branch 'jamovi:main' into weblate

sjentsch committed:
Merge pull request #1 from sjentsch/weblate

Weblate

sjentsch committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 49.5% (653 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/nb_NO/

sjentsch committed:
Translated using Weblate (German)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/de/

sjentsch committed:
Merge pull request #2 from sjentsch/weblate

Weblate

sjentsch committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 51.2% (676 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/nb_NO/

sjentsch committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/zh_Hans/

sjentsch committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/ja/

sjentsch committed:
Translated using Weblate (Spanish)

Currently translated at 98.0% (1293 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/es/

sjentsch committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/pt/

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 66.1% (873 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/sl/

MAgojam committed:
Translated using Weblate (Italian)

Currently translated at 96.3% (1271 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/it/

MAgojam committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/it/

MAgojam committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/it/

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 75.2% (992 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/sl/

Commit:
Translated using Weblate (Croatian)

Currently translated at 0.1% (2 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/hr/

MAgojam committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/it/

yurismol committed:
Added translation using Weblate (Russian)

yurismol committed:
Translated using Weblate (Russian)

Currently translated at 21.3% (281 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/ru/

MAgojam committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/it/

yurismol committed:
Translated using Weblate (Russian)

Currently translated at 37.6% (496 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/ru/

yurismol committed:
Translated using Weblate (Russian)

Currently translated at 58.4% (771 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/ru/

Commit:
Translated using Weblate (Croatian)

Currently translated at 0.2% (3 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/hr/

yurismol committed:
Translated using Weblate (Russian)

Currently translated at 59.0% (779 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/ru/

yurismol committed:
Translated using Weblate (Russian)

Currently translated at 91.1% (1202 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/ru/

urmatej committed:
Translated using Weblate (Slovenian)

Currently translated at 78.3% (1033 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/sl/

yurismol committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/ru/

jpmaroco committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/pt/

yurismol committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/ru/

sjentsch committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 57.9% (765 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/nb_NO/

sjentsch committed:
Translated using Weblate (German)

Currently translated at 100.0% (1319 of 1319 strings)

Translation: jamovi/jmv-i18n
Translate-URL: https://hosted.weblate.org/projects/jamovi/jmv-i18n/de/
